### PR TITLE
Made this a Jekyll project

### PR DESCRIPTION
**Pull request by jeffmaher**
Moved existing documents to the `docs` folder (Content and Style Guides).

This is just an empty Jekyll project, so it doesn't really do anything right now.

### CODE REVIEW

**Comment by jeffmaher** [comment]
For issue #20

File: _config.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Welcome to Jekyll!
+#
+# This config file is meant for settings that affect your whole blog, values
+# which you are expected to set up once and rarely need to edit after that.
+# For technical reasons, this file is *NOT* reloaded automatically when you use
+# 'jekyll serve'. If you change this file, please restart the server process.
+
+# Site settings
+title: Your awesome title
```
Comment (awong-dev) [inline _config.yml]: Can we add `source: _source` or something so the top-level directory containing things like .gitignores, etc., don't get mixed into the jekyll tree?

I was about to make that change for vets-website too.

Comment (plusjeff) [inline _config.yml]: Yep! Added.

Comment (awong-dev) [inline _config.yml]: oh...you need to add a jekyll_source directory too.

Comment (awong-dev) [inline _config.yml]: and then move all the jekyll template goo into it.

**Comment by emilyville** [comment]
I can't seem to run this locally - also I suggest we add a readme - this one has some basic stuff that we could port over since I think they are both stock!

https://github.com/department-of-veterans-affairs/vets.gov-status

File: _config.yml (hunk 1 above)
Comment (awong-dev) [inline _config.yml]: pushed 1a97286 to fix.

**Comment by awong-dev** [comment]
@emilyville that was due to the config change I suggested fro Jeff.  My lastest push fixes it all.

**Comment by emilyville** [comment]
LGTM!

**Pull request merged**